### DISCUSSION TITLE
Workflow works in Rstudio and fails in geoflow-shiny

### DISCUSSION THREAD

**Discussion by juliettefabre** [eblondel/geoflow-shiny#36] [Q&A] [answered]
Hello,

I just installed geoflow-shiny. I loaded a json config file which works fine when I execute it with `executeWorkflow` in Rstudio. 

When run in geoflow shiny, it fails at initialization step with this error:
`Error in geoflow::executeWorkflow(file = filepath, dir = targetdir, queue = ipc.queue, : argument inutilisé (session = parent.session) `

In the console I have the message: 
`No geoflow job started!`

Any idea?

**Comment by eblondel** [comment, marked as answer]
Your geoflow might be outdated. Update it and retry
Ps: i'm converting this to a discussion, you can rely on this for whatever question/answer
Cheers 
Le ven. 27 janv. 2023 à 14:42, juliettefabre ***@***.***> a
écrit :

**Comment by juliettefabre** [comment]
It works, thanks, I always forget to try to update packages ...^^

**Reply by eblondel** [reply]
Cool, enjoy the workshop, and good luck to @wheintz !!! Be kind with him in case you find a bug! 😁😁😁

**Reply by juliettefabre** [reply]
it's finishing!

**Reply by wheintz** [reply]
Hi @eblondel, geoflow-shiny worked perfectly, but our SK8 server was overloaded...
